Dockerfile

```diff
@@ -7,9 +7,10 @@ COPY --chown=1000:0 requirements.txt ./requirements.txt
 RUN pip install --no-cache-dir -r requirements.txt && \
     apt-get update -y && \
     apt-get install -y gettext-base apt-transport-https curl gnupg procps && \
-    curl -s https://repos.influxdata.com/influxdata-archive_compat.key | apt-key add - && \
-    . /etc/os-release && \
-    echo "deb https://repos.influxdata.com/debian stretch stable" > /etc/apt/sources.list.d/influxdb.list && \
+    # influxdata-archive_compat.key GPG Fingerprint: 9D539D90D3328DC7D6C8D3B9D8FF8E1F7DF8B07E
+    curl -s https://repos.influxdata.com/influxdata-archive_compat.key > influxdata-archive_compat.key && \
+    echo '393e8779c89ac8d958f81f942f9ad7fb82a25e133faddaf92e15b16e6ac9ce4c influxdata-archive_compat.key' | sha256sum -c && cat influxdata-archive_compat.key | gpg --dearmor | tee /etc/apt/trusted.gpg.d/influxdata-archive_compat.gpg > /dev/null && \
+    echo 'deb [signed-by=/etc/apt/trusted.gpg.d/influxdata-archive_compat.gpg] https://repos.influxdata.com/debian stable main' | tee /etc/apt/sources.list.d/influxdata.list && \
```

Comment:
Do we need the output of this command? Otherwise, just redirect it to the file without using `tee`.